lib/logstash/inputs/jdbc.rb

```diff
@@ -260,6 +270,11 @@ def register
         converters[encoding] = converter
       end
     end
+
+    # target must be populated if ecs_compatibility is not :disabled
+    if @target.nil? && ecs_compatibility != :disabled
+      logger.warn("When ECS compatibility is enabled also target option must be valued")
```

Comment:
believe we agreed upon this being logger at the `info` level (as a compromise since it might be legit to skip setting `target` in cases where data has no conflicts with the schema).

what I would be also interested, due other plugins, is a standard message we would use across plugins ... draft:
`ECS compatibility is enabled but no ``target`` option was specified, it is recommended to set the option to avoid potential schema conflicts (if you're data is ECS compliant or non-conflicting feel free to ignore this message)` // cc @yaauie @karenzone

Reply:
+1 on standardize, maybe we can put it as constant in the ECS mixin?

Reply:
@kares In another conversation for a different plugin, I believe we discussed emitting this as `info`, since it isn't exactly something that needs to be resolved.

Reply:
Can you recall maybe the plugin?So that I can use the same log info, to be consistent across various plugins

Reply:
> believe we agreed upon this being logger at the `info` level 

yes we agreed on info level logging previously, we do not have another plugin with the "warning" yet.
that's why I proposed a message for review, if accepted we would use it here and maybe move to ECS support mixing for re-use? (it's going to concern 3 other plugins I can think of right away)

Reply:
```suggestion
      logger.info("ECS compatibility is enabled but no ``target`` option was specified, it is recommended to set the option to avoid potential schema conflicts (if you're data is ECS compliant or non-conflicting feel free to ignore this message)")
```

Reply:
I think your message is perfect, in case of jdbc static and filter we would also add the `id` of the interested lookups, so in that case it could hit many lines like this, one for each lookup definition